Gemfile.lock

```diff
@@ -606,8 +606,8 @@ GEM
       unf_ext
     unf_ext (0.0.9.1)
     unicode-display_width (2.5.0)
-    view_component (2.74.1)
-      activesupport (>= 5.0.0, < 8.0)
+    view_component (2.82.0)
+      activesupport (>= 5.2.0, < 8.0)
```

Comment:
It looks like in H2, VC updated to the latest. Do you know why it was held to 2.x in Argo? Some other dependency, presumably.

Reply:
I suspect that in the early, unstable days of VC there was an update that caused a problem and we pinned a few apps and forgot to unpin.

Reply:
In this case, judging by the dependabot PR I closed, I think it is blacklight that may be holding us back.